doc/source/data/batch_inference.rst

```diff
@@ -1,823 +1,455 @@
-Just pick one of the frameworks you like and run the code in your terminal.
-If you want a more detailed rundown of the same examples, skip ahead to the
-:ref:`following batch inference walk-through with Ray<batch_inference_walk_through>`.
-
-
-.. tabs::
-
-    .. group-tab:: HuggingFace
-
-        .. literalinclude:: ./doc_code/hf_quick_start.py
-            :language: python
-            :start-after: __hf_super_quick_start__
-            :end-before: __hf_super_quick_end__
-
-    .. group-tab:: PyTorch
 
-        .. literalinclude:: ./doc_code/pytorch_quick_start.py
-            :language: python
-            :start-after: __pt_super_quick_start__
-            :end-before: __pt_super_quick_end__
+Why choose Ray Data for offline inference?
+------------------------------------------
 
-    .. group-tab:: TensorFlow
+.. dropdown:: Faster and cheaper for modern deep learning applications
 
-        .. literalinclude:: ./doc_code/tf_quick_start.py
-            :language: python
-            :start-after: __tf_super_quick_start__
-            :end-before: __tf_super_quick_end__
+    Ray Data is for deep learning applications that involve both CPU preprocessing and GPU inference. Ray Data streams the working data from CPU preprocessing tasks to GPU inferencing tasks, allowing you to utilize both sets of resources concurrently.
 
+    By using Ray Data, your GPUs are no longer idle during CPU computation, reducing overall cost of the batch inference job.
 
-.. _batch_inference_walk_through:
+.. dropdown:: Cloud, framework, and data format agnostic
 
-Walk-through: Batch Inference with Ray
---------------------------------------
+    Ray Data has no restrictions on cloud provider, ML framework, or data format.
+    
+    Through the :ref:`Ray cluster launcher <cluster-index>`, you can start a Ray cluster on any of the popular cloud providers. Since Ray Data works with any Python function or class, you can use any ML framework of your choice, including PyTorch, HuggingFace, or Tensorflow. Ray Data also does not require a particular file format, and supports a :ref:`wide variety of formats <loading_data>` including CSV, Parquet, raw images, etc.
 
-Running batch inference is conceptually easy and requires three steps:
+.. dropdown:: Out of the box scaling
 
-1. Load your data and apply any preprocessing you need.
-2. Define your model and define a transformation that applies your model to your data.
-3. Run the transformation on your data.
+    Ray Data is built on Ray, so it easily scales to many machines. The code that works on one machine also runs on a large cluster without any modifications.
 
+.. dropdown:: Python first
 
-Let's take a look at a simple example of this process without using Ray.
-In each example we load ``batches`` of data, load a ``model``, define a ``transform``
-function and apply the model to the data to get ``results``.
+    With Ray Data, you can express your inference job directly in Python instead of
```

Comment:
Nit: all of the other dropdowns start with "Ray Data is/has..." Would be nice to do the same here for consistency, but not sure how we'd reword it

Reply:
This still works? "Ray Data is Python first"